server/controllers/chartsController/chartsController.test.js

```diff
@@ -290,7 +290,7 @@ describe('chartsController', () => {
   describe('update', () => {
     describe('When successful', () => {
       it('returns a status of 200 and an updated chart', async () => {
```

Comment:
I think we should switch this test to use the test database instead of mocks, if we're in here anyway. But your call!

Reply:
💯, there are plenty of mocked tests to be updated, and I put them together in this pr with the rest of the chart query changes. https://github.com/AMP-SCZ/dpdash/pull/625